src/server/ProxiedApp.ts

```diff
@@ -51,6 +53,8 @@ export class ProxiedApp implements IApp {
 
     public makeContext(data: object): vm.Context {
         return vm.createContext(Object.assign({}, {
+            ...timers,
+            Buffer,
```

Comment:
Would be good to have a sort of `buildDefaultAppContext` that always injects these properties into the global scope of apps

Reply:
Done